### DISCUSSION TITLE
wind direction input data and output model result were different

### DISCUSSION THREAD

**Discussion by adefaizal** [NOAA-EMC/WW3#1154] [Q&A] [answered]
Hi,

I'd like to ask about the results from my model below (left wind data era5, right model output). The wind direction from the input (I used era5.nc data) and the model output were different. I thought it might be due to the layout indicator, but I've tried changing the indicator multiple times, and the result remains the same. I used an nml file instead of an inp file. Does anyone have an idea about the main issue with these results?

Thank you

https://github.com/NOAA-EMC/WW3/assets/138177594/7f42a5b5-18d9-434f-8a81-abb9bb62feb8

**Comment by aliabdolali** [comment]
I think you did not define the dimension of your input file in the right order. Please paste your ww3_prnc.inp(nml) and the header of your input netcdf (you can use ncdump for it).

**Reply by adefaizal** [reply]
Hi Ali

Thank you for your prompt response. Please find below the requested information:

prnc.nml file:

! -------------------------------------------------------------------- !
! WAVEWATCH III - ww3_prnc.nml - Field preprocessor                    !
! -------------------------------------------------------------------- !

&FORCING_NML
  FORCING%TIMESTART         = '20200801 000000'
  FORCING%TIMESTOP          = '20200930 230000'
  FORCING%FIELD%WINDS   	= t
  FORCING%GRID%ASIS     	= t
  FORCING%GRID%LATLON       = t
 /

&FILE_NML
  FILE%FILENAME      = 'koreawindaugsep2020.nc'
  FILE%LONGITUDE     = 'longitude'
  FILE%LATITUDE      = 'latitude'
  FILE%VAR(1)        = 'u10'
  FILE%VAR(2)        = 'v10'
```

``` 

! -------------------------------------------------------------------- !
! WAVEWATCH III - end of namelist                                      !
! -------------------------------------------------------------------- !


header input nc file:
<img width="944" alt="image" src="https://github.com/NOAA-EMC/WW3/assets/138177594/af62b367-4b91-4bf7-8fa3-21410b185539">


Thank youu

**Reply by aliabdolali** [reply, marked as answer]
Is your ww3 grid the same as netcdf file? It seems you defined the grid ASIS=T 
Change it to F and retry

FORCING%GRID%ASIS =F

**Reply by adefaizal** [reply]
Yeah, it's the same.

I tried to change ASIS to F and LATLON to T in ww3_prnc.nml, but it came up with this message:

<img width="496" alt="error" src="https://github.com/NOAA-EMC/WW3/assets/138177594/e08e2d92-67a9-4278-a95b-332f6aa07306">

However, when I used ww3_prep.inp with ASCII wind input format, the model result matched with the wind.

**Reply by aliabdolali** [reply]
Well, the mesage clearly says your latitude should be reversed, the. It should work. This is all about the order

**Reply by adefaizal** [reply]
Alright. Then I'll keep using prep.inp instead of prnc.nml. Thank you for your response, Ali.
